Lab_5.ipynb

```diff
@@ -0,0 +1,978 @@
+      "text/plain": [
+       "<Figure size 504x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_data('t32', '')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d5cad85-34f9-4aa3-a9d1-1475f81e13a5",
+   "metadata": {},
+   "source": [
+    "t21 and t32 appear related to the other tn distributions, which should be the case because they are ratios of one another. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf5c56eb-070c-45d8-865e-006f39e98160",
+   "metadata": {},
+   "source": [
+    "plot_data('KtDeltaR', '')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c09c841-283a-4368-9132-335c01947ea9",
+   "metadata": {},
+   "source": [
+    "KdeltaR looks similar to ee2 and ee3 distriubtions, they could be related somehow. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ee62b71-2282-4945-94cb-3dcfad65521e",
+   "metadata": {},
+   "source": [
+    "### Compute the expected discovery sensitivity (significance of the expected signal) by normalizing each sample appropriately (see 'expected yields' below)."
```

Comment:
It would help if you had these headers for the other two previous questions as well, you answer them but the responses are a little buried and difficult to find